Specs/Widgets/InfoBox/InfoBoxViewModelSpec.js

```diff
@@ -14,6 +16,13 @@ defineSuite([
         expect(viewModel.maxHeightOffset(0)).toBeDefined();
     });
 
+    it('sets title', function() {
```

Comment:
I think that was the version of the test before I changed to switching between the two views, I should probably take another look at the tests